whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,635 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
+This document summarizes all the relevant information required for the speech community event 📋.
+
+To sign-up, please see [this forum post](https://LINK_ACTUAL_FORUM) 🤗. Please make sure to:
+- Read it in detail
+- Fill the [google form](https://forms.gle/F2bpouvhDpKKisM39)
+- Join the [Hugging Face Discord server](https://hf.co/join/discord) and make sure you have access to the #events channel.
+
+[comment]: # TODO: VB - create a post on the forum and link it above. Update the URL to cloud provider when decided.
```

Comment:
To update before merge

(as mentioned above, I don't think the forum post is a requisite for this)